lib/galaxy/tools/error_reports/plugins/gitlab.py

```diff
@@ -41,7 +46,8 @@ def __init__(self, **kwargs):
         self.gitlab_labels = kwargs.get("gitlab_labels", [])
 
         try:
-            import gitlab
+            if 'gitlab' not in sys.modules:
+                raise ImportError
```

Comment:
```suggestion
            if 'gitlab' is None:
                raise Exception("gitlab error reporting plugin is configured, but gitlab is not importable")
```
Or something like this.